nanome/util/asyncio.py

```diff
@@ -1,8 +1,41 @@
 import asyncio
+import logging
+import sys
+import nanome
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    """Make sure uncaught exceptions are logged."""
+    if issubclass(exc_type, KeyboardInterrupt):
+        sys.__excepthook__(exc_type, exc_value, exc_traceback)
+        return
+
+    logger_name = nanome._internal.LOGGER_NAME
+    if logger_name:
+        logger = logging.getLogger(logger_name)
+    else:
+        logger = logging.getLogger()
+
+    msg = "Uncaught " + exc_type.__name__ + ": " + str(exc_value)
+    logger.error(msg, exc_info=1)
+
+
+@asyncio.coroutine
+def exception_wrapper(fn, args, kwargs):
```

Comment:
I believe this can be replaced with `async def`, and the `yield from` later with `await`